apps/agent/src/main/workers/push-activities.ts

```
@@ -16,12 +16,15 @@ class PushActivities {
 	private apiService = ApiService.getInstance();
 	private agentLogger: AgentLogger;
 	private mainEvent: MainEvent;
+	private isNetworkError: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)